src/main/java/macrobase/util/DiagnosticsUtils.java

```diff
@@ -9,19 +9,14 @@
 
 public class DiagnosticsUtils {
 
-    public static List<Datum> create2DGrid(double[][] boundaries, double delta) {
+    public static List<Datum> createGridFixedIncrement(double[][] boundaries, double delta) {
```

Comment:
I want to actually see the distribution in areas without data, also it makes plotting much easier.